eisbuk-admin/public/locales/en/translation.json

```diff
@@ -1,118 +1,113 @@
 {
-    "AppbarAdmin": {
-      "Attendance": "Attendance",
-      "Athletes": "Athletes",
-      "Reservations": "Reservations"
-    },
-    "Slots": {
-        "Amateur": "Amateur",
-        "DeleteConfirmation": "Are you sure you want to delete the ",
-        "All":"",
-        "NonReversible": "This action is non-reversible",
-        "Slot": "slot",
-        "Book":"Book",
-        "Reserved": "Reserved",
-        "ConfirmDialogDate":"{{date, d MMMM}}",
-        "ConfirmDialogTime":"{{date, HH:mm}}"
-    },
-    "SlotValidations":{
-        "Time": "Select the time",
-        "Category":"Choose at least one category",
-        "Training": "Choose the type of training",
-        "Duration": "Choose at least one duration",
-        "Required": "Required"
+  "AppbarAdmin": {
+    "Attendance": "Attendance",
+    "Athletes": "Athletes",
+    "Reservations": "Reservations"
+  },
+  "Slots": {
+    "Amateur": "Amateur",
+    "DeleteConfirmation": "Are you sure you want to delete the ",
+    "All": "",
+    "NonReversible": "This action is non-reversible",
+    "Slot": "slot",
+    "Book": "Book",
+    "Reserved": "Reserved",
+    "ConfirmDialogDate": "{{date, d MMMM}}",
+    "ConfirmDialogTime": "{{date, HH:mm}}"
+  },
+  "SlotValidations": {
+    "Time": "Select the time",
+    "Category": "Choose at least one category",
+    "Training": "Choose the type of training",
+    "Duration": "Choose at least one duration",
+    "Required": "Required"
+  },
+  "SlotForm": {
+    "StartTime": "Start Time",
+    "Type": "Type",
+    "EditSlot": "Edit Slot",
+    "CreateSlot": "Create Slot",
+    "Cancel": "Cancel",
+    "parsedSlotEditDate": "{{date, EEEE d MMMM - HH:mm}}",
+    "parsedDate": "{{date, EEEE d MMMM}}"
+  },
+  "CustomerForm": {
+    "Name": "Name",
+    "Surname": "Surname",
+    "Phone": "Phone",
+    "DateOfBirth": "Date Of Birth",
+    "Category": "Category",
+    "MedicalCertificate": "Medical Certificate Expiration",
+    "CardNumber": "Card Number",
+    "Cancel": "Cancel",
+    "Save": "Save",
+    "NewAthlete": "New Athlete"
+  },
+  "CustomerValidations": {
+    "Email": "Please insert a valid email",
+    "Category": "Select a category",
+    "Required": "Required"
+  },
+  "Authorization": {
+    "NotAuthorized": "You are not authorized to login",
+    "AdminsOnly": "This space is reserved for administrators",
+    "LoggedInWith": "You are logged in with: ",
+    "TryAgain": "Log out and try again with a different user"
+  },
+  "CustomerArea": {
+    "book_ice": "Book Ice",
+    "book_of_ice": "Book Off-Ice",
+    "calendar": "Calendar"
```

Comment:
Please note these: The enums (when they're compiled) are values for each `Enum.Key`. This is why the keys here are written the same as enum values of said "routes".